redwood-widget-compose/src/commonMain/kotlin/app/cash/redwood/widget/compose/ComposeWidgetChildren.kt

```diff
@@ -16,41 +16,41 @@
 package app.cash.redwood.widget.compose
 
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateListOf
-import app.cash.redwood.LayoutModifier
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
 import app.cash.redwood.widget.Widget
 
 public class ComposeWidgetChildren : Widget.Children<@Composable () -> Unit> {
-  private val layoutModifiers = mutableStateListOf<LayoutModifier>()
+  private var layoutModifierTick by mutableStateOf(0)
 
   private val _widgets = mutableStateListOf<Widget<@Composable () -> Unit>>()
   public val widgets: List<Widget<@Composable () -> Unit>> get() = _widgets
 
   @Composable
   public fun render() {
+    // Observe the layout modifier count so we recompose if it changes.
+    layoutModifierTick
+
     for (index in _widgets.indices) {
-      // Observe the layout modifier so we recompose if it changes.
-      layoutModifiers[index]
       _widgets[index].value()
     }
```

Comment:
Should have the same behavior. We dirty the nodes produced by this loop which forces the parent into which these nodes are placed to relayout. Would love to figure out how to replace this implicit relationship with one that's more explicit.

Reply:
Tricky without making a `ComposeableWidget` class which produces snapshot changes.